src/analytics/provider/analytics.state.ts

```diff
@@ -0,0 +1,36 @@
+import { Trade } from '@swapr/sdk'
+
+interface AnalyticsTradeQueueItem {
+  id: string
+  trade: Trade
+  status: 'pending' | 'completed'
```

Comment:
We also should define this keys inside an object :)

Reply:
Good point :)